docs/integration_guide.md

```diff
@@ -26,31 +26,132 @@ The order of events for relaying transactions or deploying smart wallets through
 Another option is to use Enveloping through a Relay Provider. The latter wraps web3, and then all transactions and calls are made through the Relay Provider. If a Relay Client is not provided then the Relay Provider creates an instance.
 
 ```typescript
-    this.config = await resolveConfiguration(web3.currentProvider, {
-      verbose: window.location.href.includes('verbose'),
-      onlyPreferredRelays: false, //If false it will look for a relayer, if true it reads preferred Relays
-      chainId: chainId,
-      relayVerifierAddress: this.contracts.relayVerifier,
-      factory: this.contracts.factory,
-      preferredRelays:[]
-    })
-
-    this.provider = new Enveloping.RelayProvider(web3.currentProvider, this.config)
-    web3.setProvider(this.provider)
-
-    this.provider.deploySmartWallet(trxData)
-    this.provider.calculateSmartWalletAddress(
-    factory.address,gaslessAccount.address, recoverer, customLogic, walletIndex, bytecodeHash)
-
-    await testRecipient.emitMessage('hello world', {
-        from: gaslessAccount.address,
-        gas: '100000',
-        gasPrice: '1',
-        callVerifier: verifierInstance.address,
-        onlyPreferredRelays: true //It will read the preferredRelays on the config.
-    })
+    import { RelayProvider, resolveConfiguration } from "@rsksmart/enveloping";
+    import Web3 from "web3";
+
+    const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+    const web3 = new Web3("http://localhost:4444");
+    
+    const smartWalletFactoryAbi = {};// some json containing the abi of the smart wallet factory contract.
+    const smartWalletFactoryAddress = "0x3bA95e1cccd397b5124BcdCC5bf0952114E6A701"; // the smart wallet factort contract address (can be retrieved from the summary of the deployment).
+    const smartWalletIndex = 0; // the index of the smart wallet
+    const accountAddress = "0xB6009fD7a09cB4C6846ab19Be81d7D0b533cF8B7"; // your account address
+    const accountPrivateKey = "0xc2decf57baed5c1f4e546dc04eab97edb753c72723720af95f14c3ac1e3ef830"; // your account private key
```

Comment:
Yes we know but to do that outside enveloping you need to have the AccountManager exported and it's not so you can't import it to instantiate it, so based on that we can't do it because is impossible to get the definition from the enveloping library, that's something we should improve (exporting resources) anyway, but we know is not a good idea this was just a working example.

Reply:
I suggest to fix the export problem, or at least to show a snippet of code with the potential solution instead of suggesting the usage of the private key in this way. If you take a look at the demo we created for the webinar, it is possible to import the different Enveloping components from the dist folder once it is built. The only missing part is the creation and publishing of an npm  package to make it easier to be imported.

Reply:
Make sense, i will add an external reference and put a note to let the user know that it should be externally managed instead of putting the privateKey directly